cache/client.go

```diff
@@ -35,47 +34,50 @@ type clientMetrics struct {
 	dataSize   *prometheus.HistogramVec
 }
 
-func newClientMetrics(reg prometheus.Registerer) *clientMetrics {
-	cm := &clientMetrics{}
-
-	cm.operations = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+func newClientMetrics(regs []prometheus.Registerer) *clientMetrics {
+	//lint:ignore faillint need to apply the metric to multiple registerer
```

Comment:
I think it is OK to ignore the lint rule in this case. `promauto` is meant for "simple" cases. And registering the same metric with different registerers is a prime example for those rare "involved" cases for which `promauto` isn't enough.

Reply:
Now you can use `promauto` again, can't you?

Reply:
I should have realized this. Updated, now everything use promauto again.